CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Releases
 v1.10.0 (unreleased)
 -------------------
 
+-   Thrift: UberFx-compatible modules are now generated for each service inside
+    a subpackage. Disable this by passing a `-no-fx` flag to the plugin.
```

Comment:
I'm mostly indifferent to the default here, as long as this is on by default in our internal go-build.

On-by-default in OSS seemed okay to me since it doesn't introduce any non-test Fx dependencies in YARPC, and only forces users to depend on Fx if they're using the generated Fx package. I'm also fine with making this an internal plugin that's auto-installed by go-build.